### PR TITLE
Change header props in connection data table

### PR DESCRIPTION
**Pull request by sccofield**
Change the header props in connection data table from accepting only an array of strings to being able to accept an array of objects.

Closes [#1597](https://github.com/taskcluster/taskcluster/issues/1597)

### CODE REVIEW

File: ui/src/components/ClientsTable/index.jsx [hunk 1]
```diff
@@ -31,7 +34,7 @@ export default class ClientsTable extends Component {
   };
 
   state = {
-    sortBy: tableHeaders[0],
+    sortBy: 'client id',
     sortDirection: 'asc',
   };
```
Comment (helfi92) [inline ui/src/components/ClientsTable/index.jsx]: Navigating to http://localhost:5080/auth/clients shows:

Screen Shot 2019-10-23 at 2 57 29 PM

On the production site, there's an arrow beside the client id column name:
Screen Shot 2019-10-23 at 2 57 39 PM

**Comment by helfi92** [comment]
@all-contributors please add @sccofield for code.

**Comment by helfi92** [comment]
How is it going @sccofield? Let me know if you had any questions.

**Comment by djmitche** [comment]
(closing as stale)

**Pull request closed without merging**